scripts/code.angularjs.org-firebase/readme.firebase.code.md

```diff
@@ -0,0 +1,10 @@
+Firebase for code.angularjs.org
+===============================
+
+This folder contains the Google Firebase scripts for the code.angularjs.org setup.
+
+firebase.json contains the rewrite rules that route every subdirectory request to the cloud function
+in functions/index.js that serves the docs from the Firebase Google Cloud Storage bucket.
+
+The deployment to the Google Cloud Storage bucket happens automatically via Travis. See the travis.yml
+file in the repository root.
```

Comment:
Could be worth a link to the other doc here, just in case people come across this file when they are looking for the other firebase configuration.